documentation/breaking-changes/upcoming-breaking-changes.md

```diff
@@ -25,5 +25,106 @@
     https://github.com/Azure/azure-powershell/blob/dev/documentation/breaking-changes/breaking-change-template.md
 -->
 
-# Upcoming Breaking Changes
+# Upcoming Breaking Changes of Az.DesktopVirtualization
 
```

Comment:
Could you remove the file as the content is moved to the new PR https://github.com/Azure/azure-powershell/pull/26269